rav1e_js/Cargo.toml

```diff
@@ -14,17 +14,30 @@ crate-type = ["cdylib", "rlib"]
 default = ["console_error_panic_hook"]
 
 [dependencies]
+dcv-color-primitives = { git = "https://github.com/aws/dcv-color-primitives", branch = "wip/wasm" }
```

Comment:
Usually we do not use WIP/Experimental in master, 

I can see open ticket in the issue tracker, maybe can we have a release patch release in upstream soonish? Can you please see with them, so it will be easy for us.

Else we might need to have local crate, 

Not sure if we could use wip/exp branch version from git here.

Reply:
They released it in `0.1.14` (https://crates.io/crates/dcv-color-primitives) :blush: